### PR TITLE
sum, sort-strings, pick and omit tasks are done

### PR DESCRIPTION
**Pull request by boleque**
Intro task 1-sum is done

### CODE REVIEW

File: 01-intro/1-sum/index.js [hunk 1]
```diff
@@ -5,5 +5,5 @@
  * @returns {number}
  */
 export default function sum(m, n) {
-
+    return m + n;
```
Comment (dosandk) [inline 01-intro/1-sum/index.js:8]: good 👍

File: 02-javascript-data-types/1-sort-strings/index.js [hunk 2]
```diff
@@ -6,4 +6,10 @@
  */
 export function sortStrings(arr, param = 'asc') {
 
-}
+    return [...arr].sort(
+        function(a, b){
+            let ret = a.localeCompare(b, 'ru-u-kf-upper');
```
Comment (dosandk) [inline 02-javascript-data-types/1-sort-strings/index.js:11]: проверьте, а точно `let`?

**Review by dosandk — APPROVED** [review]
good 👍

File: 02-javascript-data-types/1-sort-strings/index.js (hunk 2 above)
Comment (boleque) [reply 02-javascript-data-types/1-sort-strings/index.js:11]: Да, const уместнее.

**Pull request merged**